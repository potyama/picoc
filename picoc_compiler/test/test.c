int main()
{
	int	i;

	i = - - 100;
	write(i);
	writeln();
}
