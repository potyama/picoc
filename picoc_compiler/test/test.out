#Picoc09 2011-12-13
__start__:
	call	main
	halt
#組込み関数 read()
#ここに read() の機械語を書く
read:
	enter
	rd
	storel	2
	leave
	ret
#組込み関数 write()
#ここに write() の機械語を書く
write:
	enter
	pushl	2
	wr
	leave
	ret
#組込み関数 writeln()
writeln:
	enter
	wrln
	leave
	ret
main:
	enter
	mvsp	-1
	pushi	10
	storel	-1
